From 9af753c46896cfcb5480bbb1dd896e042043349c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 9 Jul 2015 16:14:50 +0200
Subject: [PATCH] crypt: install drbg unconditionally in hostonly mode

older kernels had the drbg kernel module and didn't need it

(cherry picked from commit 89948e58fd0f80def0912c18c503912873aa9c48)
---
 modules.d/90crypt/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90crypt/module-setup.sh b/modules.d/90crypt/module-setup.sh
index e8c690e..4df520c 100755
--- a/modules.d/90crypt/module-setup.sh
+++ b/modules.d/90crypt/module-setup.sh
@@ -25,6 +25,7 @@ depends() {
 # called by dracut
 installkernel() {
     instmods dm_crypt =crypto
+    hostonly="" instmods drbg
 }
 
 # called by dracut
